crm-project/src/main/java/CRM/service/TypeService.java

```diff
@@ -2,17 +2,30 @@
 
 import CRM.controller.facades.AttributeFacade;
 import CRM.entity.Attribute;
+import CRM.entity.Status;
+import CRM.entity.Type;
+import CRM.repository.TypeRepository;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class TypeService implements AttributeService{
 
     private static Logger logger = LogManager.getLogger(TypeService.class.getName());
 
+    @Autowired
+    private TypeRepository typeRepository;
+
```

Comment:
I know it's a small edit, but as a convention, we separate between code blocks with only one line, so remove one please :)

Reply:
already fixed that